lib/rasn1/types/bmp_string.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module RASN1
+  module Types
+    # ASN.1 BmpString
+    class BmpString < OctetString
+      # BmpString id value
+      ID = 30
+
+      # Get ASN.1 type
+      # @return [String]
+      def self.type
+        'BmpString'
+      end
+
+      private
+
+      def value_to_der
+        @value.to_s.dup.encode('UTF-16BE').b
+      end
+
+      def der_to_value(der, ber: false)
+        super
+        @value.to_s.dup.force_encoding('UTF-16BE')
+      end
```

Comment:
This follows the pattern from IA5String, but I'm wondering if this should be:

```suggestion
      def der_to_value(der, ber: false)
        @value = der.to_s.dup.force_encoding('UTF-16BE')
      end
```

It doesn't seem right to me that this test would expect `.value` to be ASCII-8BIT instead of UTF16-BE - but let me know your thoughts 💯 

```
    it 'parses a DER BMPString' do
      subject.parse!("\x1e\x02\x03\x91".b)
      expect(subject.value).to eq("\x03\x91".b)
      expect(subject.value.encoding).to eq(Encoding::ASCII_8BIT)
    end
```